src/Tizen.NUI.Components/Utils/StyleManager.cs

```diff
@@ -23,7 +24,8 @@ namespace Tizen.NUI.Components
     /// <summary>
     /// StyleManager is a class to manager all style.
     /// </summary>
-    /// <since_tizen> 8 </since_tizen>
+    /// <since_tizen> none </since_tizen>
```

Comment:
Please don't use `<since_tizen> none </since_tizen>` tag, because it is not an appropriate tag value. If you hide this API by using EditorBrowsableState.Never you can remove `<since_tizen>` tag completely or just leave it as it is.
```suggestion
    /// <since_tizen> 8 </since_tizen>
```
```suggestion
```

Reply:
Done!